tests/testthat/test-compat_friendly_type.R

```diff
@@ -1,11 +1,12 @@
+# friendly_type_of ----
 ## Test 1: friendly_type_of() supports objects ----
-test_that("compat_friendly_type Test 1: friendly_type_of() supports objects", {
+test_that("friendly_type_of Test 1: friendly_type_of() supports objects", {
   expect_equal(friendly_type_of(mtcars), "a <data.frame> object")
   expect_equal(friendly_type_of(quo(1)), "a <quosure> object")
 })
 
 ## Test 2: friendly_type_of() supports matrices and arrays (#141) ----
-test_that("compat_friendly_type Test 2: friendly_type_of() supports matrices and arrays (#141)", {
+test_that("friendly_type_of Test 2: friendly_type_of() supports matrices and arrays (#141)", {
```

Comment:
Do you know what this #141 is?  This might be from the rlang package.  I copied some of these tests over from them